### ISSUE TITLE
Room directory protocol listing fails

### ISSUE THREAD

**Issue by Half-Shot** [Half-Shot/matrix-appservice-discord#415]
The logging shows a 404, so it feels like something isn't hooked properly:
```
Mar-15 02:08:05.631 [DiscordAS] verbose: matrix-appservice-bridge ::1 - - [15/Mar/2019:01:08:05 +0000] "GET /_matrix/app/unstable/thirdparty/protocol/discord HTTP/1.1" 404 187 "-" "Synapse/0.99.2"
```

**Comment by Skealz** [comment]
Does this issue prevent the bridge to work ? I'm trying to set it up and I see this error. I am not able to join #_discord_* rooms but I'm not sure it is linked to this issue.

**Comment by auscompgeek** [comment]
@Skealz No, this only affects the third-party protocol list in the room directory.

**Comment by Skealz** [comment]
So, I am still supposed to be able to join rooms through the Room directory ? if not, how do I join them ?

**Comment by FSG-Cat** [comment]
Just to update this issue it seems that this bug now also causes the whole Room Directory function in Element to completely colapse upon it self.
